### PR TITLE
Env setup

### PR DESCRIPTION
**Pull request by maronuu**
## What
Set up environments and github workflows, and add library srcs.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
```
Comment (kitagry) [inline pyproject.toml:1]: You should install dynamic-versioning for publish pypi.

https://github.com/m3dev/gokart/blob/master/pyproject.toml#L13-L16

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+name = "kannon"
+version = "0.1.0"
+description = ""
+authors = ["yutaro-oguri <yutaro-oguri@m3.com>"]
```
Comment (kitagry) [inline pyproject.toml:5]:
```suggestion
authors = ["M3, inc."]
```

File: kannon/master.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+            sleep(1.0)
+            if not self._is_executable(task):
+                task_queue.append(task)
+                continue
+            # execute task
+            if isinstance(task, TaskOnBullet):
+                logger.info(
+                    f"Trying to run task {self._gen_task_info(task)} on child job..."
+                )
+                self._exec_bullet_task(task)
```
Comment (kitagry) [inline kannon/master.py:77]:
```suggestion
                self._exec_kannon_task(task)
```

File: pyproject.toml (hunk 1 above)
Comment (maronuu) [reply pyproject.toml:1]: Added in ba418e5 .

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "kannon"
+version = "0.1.0"
+description = ""
```
Comment (kitagry) [inline pyproject.toml:4]: Could you write description?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# kannon
+
+Kannon is a wrapper for the [Gokart](https://github.com/m3dev/gokart) library that allows gokart tasks to be easily executed in a distributed and parallel manner on multiple [kubernetes](https://kubernetes.io/) jobs.
```
Comment (hirosassa) [inline README.md:3]: There exists `Gokart` and `gokart` in a document. It should be consistent

**Review by hirosassa — COMMENTED** [review]
I added a minor comment.

File: kannon/kube_util.py [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    api_instance: client.BatchV1Api, job: client.V1Job, namespace: str
+) -> None:
+    api_response = api_instance.create_namespaced_job(
+        body=job,
+        namespace=namespace,
+    )
+    logger.debug(f"Job created. status={api_response.status}")
+
+
+def wait_job_until_completion(
```
Comment (kitagry) [inline kannon/kube_util.py:28]: Is this function used?

**Review by kitagry — APPROVED** [review]
LGTM except for some comments.

File: kannon/kube_util.py (hunk 6 above)
Comment (maronuu) [reply kannon/kube_util.py]: Not used, it is a function for debugging.
It is removed in commit: ee35c30

File: pyproject.toml (hunk 4 above)
Comment (maronuu) [reply pyproject.toml]: Supported in commit: 90d9c14 .

**Review by hirosassa — APPROVED** [review]
LGTM!

**Pull request merged**